### PR TITLE
forbid null string in postgresql

### PR DESCRIPTION
**Pull request by willrnch**
We force Postgres strings to be not null because golang cannot scan a null string to a go String.

### CODE REVIEW

File: modules/users/lib/users/users.go [hunk 1]
```diff
@@ -328,32 +328,20 @@ func (u *Users) GetUser(id string) (*nano.User, error) {
...
+			&user.FirstName,
+			&user.LastName,
+			&user.Email,
 			&user.IsAdmin,
 			&user.Activated,
-			&sam,
-			&windowsPassword,
+			&user.Sam,
+			&user.WindowsPassword,
 		)
```
Comment (Gentux) [inline modules/users/lib/users/users.go:340]: I don't understand what this have to do with this commit.

Can you make another commit for these changes ?

**Comment by Gentux** [comment]
2 things before accepting this requests : 
- Separate change in 2 commit
- Make commit message more accurate (and begin with uppercase letter)

File: modules/users/lib/users/users.go (hunk 1 above)
Comment (willrnch) [inline modules/users/lib/users/users.go:340]: As we don't have null strings in Pg anymore, we don't need to use `sql.NullString` so we can `Scan` directly to `*String`.
It fairly belongs to this commit.
